ui/oopsyraidsy/data/05-shb/raid/e7n.js

```diff
@@ -66,22 +66,20 @@
       id: 'E7N Light\'s Course',
       damageRegex: ['4C3E', '4C40', '4C22', '4C3C', '4E63'],
       condition: function(e, data) {
-        return !data.hasUmbral[e.targetName];
+        return data.hasAstral[e.targetName];
       },
       mistake: function(e) {
-        if (data.hasAstral[e.targetName])
-          return { type: 'fail', blame: e.targetName, text: e.abilityName + ' wrong buff' };
-        return { type: 'warn', blame: e.targetName, text: e.abilityName + ' no buff' };
+        return { type: 'fail', blame: e.targetName, text: e.abilityName + ' wrong buff' };
       },
     },
     {
       id: 'E7N Darks\'s Course',
       damageRegex: ['4C3D', '4C23', '4C41', '4C43'],
       condition: function(e, data) {
-        return data.hasUmbral[e.targetName];
+        return !data.hasAstral[e.targetName];
       },
       mistake: function(e) {
-        return { type: 'fail', blame: e.targetName, text: e.abilityName + ' wrong buff' };
+        return { type: 'fail', blame: e.targetName, text: e.abilityName + ' wrong/no buff' };
```

Comment:
I'm not sure I understand all the changes here? What was wrong with how it was before?

Reply:
Ack. I think I pulled this from a development branch instead of Master. I'm so sorry.